index.html

```diff
@@ -41,14 +43,17 @@
                 <form>
                     <div class="input-field">
                         <input class="search" id="msearch" type="search" required>
-                        <label for="msearch"><i class="material-icons"><i class="fas fa-cocktail"></i>search</label>
+                        <label for="msearch"><i class="material-icons fas fa-cocktail"></i>search</label>
                         <i class="material-icons">close</i>
                     </div>
                 </form>
             </li>
-            <li><a href="#">Random Cocktail</a></li>
+            <li><p id="newRandomDrink">Random Cocktail</p></li>
             <li><a href="#">Advanced Search</a></li>
```

Comment:
Replace Advance Search with Look up by Alcohol type or something along the lines.